### PR TITLE
Adds missing plugins to page and adds information for each plugin

### PR DESCRIPTION
**Pull request by theg00s3**
Adds all the missing plugins to the plugins page, and adds some information about each (gleaned from the descriptions in Jellyfin's plugins tab)

### CODE REVIEW

**Comment by theg00s3** [comment]
The new capitalisation and spacing doesn't throw any linking errors in docfx

File: general/server/plugins/index.md [hunk 1]
```diff
@@ -33,61 +33,157 @@ All plugins hosted on the repository can be built from source and manually added
...
+Manage your Anime in Jellyfin. Supports several different metadata providers and options for organizing your collection.
+
+#### Auto Organize
 
 [![](https://img.shields.io/github/languages/top/jellyfin/jellyfin-plugin-autoorganize.svg)](https://github.com/jellyfin/jellyfin-plugin-autoorganize)
 [![](https://img.shields.io/github/contributors/jellyfin/jellyfin-plugin-autoorganize.svg)](https://github.com/jellyfin/jellyfin-plugin-autoorganize)
 [![](https://img.shields.io/github/license/jellyfin/jellyfin-plugin-autoorganize.svg)](https://github.com/jellyfin/jellyfin-plugin-autoorganize)
 
-#### emailnotifications
+Automatically organize your media
```
Comment (dkanada) [inline general/server/plugins/index.md]: I'd use full sentences for all the descriptions.

File: general/server/plugins/index.md [hunk 2]
```diff
@@ -33,61 +33,157 @@ All plugins hosted on the repository can be built from source and manually added
...
-#### iptv
+Send SMTP email notifications
+
+#### Fanart
+
+[![](https://img.shields.io/github/languages/top/jellyfin/jellyfin-plugin-fanart.svg)](https://github.com/jellyfin/jellyfin-plugin-fanart)
+[![](https://img.shields.io/github/contributors/jellyfin/jellyfin-plugin-fanart.svg)](https://github.com/jellyfin/jellyfin-plugin-fanart)
+[![](https://img.shields.io/github/license/jellyfin/jellyfin-plugin-fanart.svg)](https://github.com/jellyfin/jellyfin-plugin-fanart)
+
+Scrape poster images for movies, shows, and artists in your library from [fanart.tv](fanart.tv)
```
Comment (dkanada) [inline general/server/plugins/index.md]: Use HTTPS here so we can gain some benefit from using markdown for the link.

File: general/server/plugins/index.md [hunk 3]
```diff
@@ -33,61 +33,157 @@ All plugins hosted on the repository can be built from source and manually added
...
-#### ldapauth
+Enable IPTV support in Jellyfin
+
+#### Kodi Sync Queue
+
+[![](https://img.shields.io/github/languages/top/jellyfin/jellyfin-plugin-kodisyncqueue.svg)](https://github.com/jellyfin/jellyfin-plugin-kodisyncqueue)
+[![](https://img.shields.io/github/contributors/jellyfin/jellyfin-plugin-kodisyncqueue.svg)](https://github.com/jellyfin/jellyfin-plugin-kodisyncqueue)
+[![](https://img.shields.io/github/license/jellyfin/jellyfin-plugin-kodisyncqueue.svg)](https://github.com/jellyfin/jellyfin-plugin-kodisyncqueue)
+
+Sync queues to Kodi
```
Comment (dkanada) [inline general/server/plugins/index.md]: Something more descriptive here might be useful, @mcarlton00 could you think of a better description?

File: general/server/plugins/index.md [hunk 4]
```diff
@@ -33,61 +33,157 @@ All plugins hosted on the repository can be built from source and manually added
...
+
+Provides access to LiveTV, Program Guide and Recordings from NextPVR
+
+#### [Open Subtitles](xref:server-plugins-open-subtitles)
+
+[![](https://img.shields.io/github/languages/top/jellyfin/jellyfin-plugin-opensubtitles.svg)](https://github.com/jellyfin/jellyfin-plugin-opensubtitles)
+[![](https://img.shields.io/github/contributors/jellyfin/jellyfin-plugin-opensubtitles.svg)](https://github.com/jellyfin/jellyfin-plugin-opensubtitles)
+[![](https://img.shields.io/github/license/jellyfin/jellyfin-plugin-opensubtitles.svg)](https://github.com/jellyfin/jellyfin-plugin-opensubtitles)
+
+Download subtitles from the internet to use with your media files from [OpenSubtitles.org](https://www.opensubtitles.org/)
```
Comment (dkanada) [inline general/server/plugins/index.md]:
```suggestion
Download subtitles from the internet to use with your media files from [Open Subtitles](https://www.opensubtitles.org/)
```

File: general/server/plugins/index.md [hunk 5]
```diff
@@ -33,61 +33,157 @@ All plugins hosted on the repository can be built from source and manually added
...
+
+#### Playback Reporting
+
+[![](https://img.shields.io/github/languages/top/jellyfin/jellyfin-plugin-playbackreporting.svg)](https://github.com/jellyfin/jellyfin-plugin-playbackreporting)
+[![](https://img.shields.io/github/contributors/jellyfin/jellyfin-plugin-playbackreporting.svg)](https://github.com/jellyfin/jellyfin-plugin-playbackreporting)
+[![](https://img.shields.io/github/license/jellyfin/jellyfin-plugin-playbackreporting.svg)](https://github.com/jellyfin/jellyfin-plugin-playbackreporting)
+
+Collect and show user play stats
```
Comment (dkanada) [inline general/server/plugins/index.md]:
```suggestion
Collect and show user playback statistics.
```

File: general/server/plugins/index.md [hunk 6]
```diff
@@ -33,61 +33,157 @@ All plugins hosted on the repository can be built from source and manually added
...
+Automatically create movie box sets based on TMDb collections
+
+#### Trakt
 
 [![](https://img.shields.io/github/languages/top/jellyfin/jellyfin-plugin-trakt.svg)](https://github.com/jellyfin/jellyfin-plugin-trakt)
 [![](https://img.shields.io/github/contributors/jellyfin/jellyfin-plugin-trakt.svg)](https://github.com/jellyfin/jellyfin-plugin-trakt)
 [![](https://img.shields.io/github/license/jellyfin/jellyfin-plugin-trakt.svg)](https://github.com/jellyfin/jellyfin-plugin-trakt)
 
-#### tvheadend
+Record your watched media with Trakt.
```
Comment (dkanada) [inline general/server/plugins/index.md]: Maybe add a link to Trakt here?

File: general/server/plugins/index.md (hunk 3 above)
Comment (mcarlton00) [reply general/server/plugins/index.md]: Something along the lines of
* "Helps keep Jellyfin for Kodi in sync with the library without needing to run periodic full scans"
or
* "Eliminates the need to run periodic full scans when using the Jellyfin for Kodi addon"

File: general/server/plugins/index.md (hunk 6 above)
Comment (theg00s3) [reply general/server/plugins/index.md]: done

File: general/server/plugins/index.md (hunk 1 above)
Comment (theg00s3) [reply general/server/plugins/index.md]: I’ll probably need some help with this, as I don’t actually know what some of these plugins do.

**Comment by theg00s3** [comment]
Should the GitHub links for the official plugins be added?

File: general/server/plugins/index.md (hunk 1 above)
Comment (dkanada) [reply general/server/plugins/index.md]: It's fine, we can always do this in another pull request.

**Comment by dkanada** [comment]
It might be a nice addition.

**Comment by theg00s3** [comment]
All done

**Pull request merged**